src/pages/api/verification/create.js

```diff
@@ -0,0 +1,55 @@
+import { getServerSession } from "@dbcdk/login-nextjs/server";
```

Comment:
/** @file ... this file creates something .. beskrivels af hvorfor/hvad denher file er/gør